src/encoded/server_defaults.py

```diff
@@ -24,8 +24,16 @@ def includeme(config):
     config.registry[ACCESSION_FACTORY] = factory
 
 
+# XXX: This stuff is all added based on the serverDefault identifier in the schemas
+# removing it altogether will totally break our code
+
+
 @server_default
 def userid(instance, subschema):
+    return _userid(instance, subschema)
+
+
+def _userid(instance, subschema):
```

Comment:
Don't you want to make the internal `_userid` function be a function of no arguments? The `instance` and `subschema` seem an artifact of `@server_default`, not of asking for a userid, unless I'm misunderstanding something. That would make the uses of this cleaner, such as `_userid()` rather than `_userid(None, None)`.